getSigmaPointApp.m

```diff
@@ -201,6 +218,88 @@
         % Weights
         w_m = 1/(size(SP.B_SP,2))*ones(size(SP.B_SP,2),1);
         w_c = 1/((size(SP.B_SP,2))-1)*ones(size(SP.B_SP,2),1);
+    case 'halton'    % Halton Monte Carlo sequence
+        samplescdf = net(haltonset(2,'skip',100),op_SP.n_samples);
+        op_SP.samples = norminv(samplescdf,0,1);
+        SP.B_SP = transpose(op_SP.samples*S);
```

Comment:
Did you check if the samples where really distributed as they should with D?
We have/had some issues there...
But this should be part of another PR, actually...